cmd/localCI/README.rst

```diff
@@ -21,10 +21,13 @@ Logging
 -------
 
 By default **localCI** places all the logs generated by each stage in ``/var/log/localCI/`` inside a directory with the
-same number of the pull request, but you can specify a different path modifying ``logDir`` in the `Configuration file`_.
-One log file per stage is generated with the same name. For example, if the ``logDir`` is
-``/var/log/localCI`` and the pull request being tested is the number ``53``, when the stage ``setup`` is run
-the full path to its log file would be ``/var/log/localCI/53/setup``.
+same number of the pull request or the master branch SHA, but you can specify a different path modifying
+``logDir`` in the `Configuration file`_. One log file per stage is generated with the same name.
+For example, if the ``logDir`` is ``/var/log/localCI`` and the pull request being tested is the number ``53``,
+when the stage ``setup`` is run the full path to its log file would be ``/var/log/localCI/53/setup``.
+Another example is when the master branch is being tested, using ``/var/log/localCI/runtime`` as ``logDir``
```

Comment:
To be consistent across our two examples, consider:
Alternatively, if  the ``logDir`` is  ``/var/log/localCI/runtime`` and the master branch is being tested, when the stage ``run`` is run, the full path to it's log file would depend on the master branch SHA and be similar to ``/var/log/localCI/runtime/a87b635cc5fg0/run``.